readme.md

```diff
@@ -81,29 +64,29 @@ If you want to modify or contribute changes to the SDK, then you can build the l
 # How to Contribute
 
 If you encounter any bugs, have suggestions for new features or if you would like to become an active contributor to this project please follow the instructions provided in the [contribution guidelines](.github/CONTRIBUTING.md).
-- **Have a feature request for SDKs?** Please post it on [User Voice](https://feedback.azure.com/forums/321918-azure-iot) to help us prioritize
 - **Have a technical question?** Ask on [Stack Overflow with tag "azure-iot-hub"](https://stackoverflow.com/questions/tagged/azure-iot-hub)
```

Comment:
Can we remove the StackOverflow thing too? We don't really monitor that. GitHub issues are better.